randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1261 @@ Ruins Entrance
 
 > Dock to Transport to Magmoor Caverns North; Heals? False
   * Normal Door to Transport to Magmoor Caverns North/Dock to Transport Access North
+  > Dock to Hive Totem
+      Morph Ball Bomb and Morph Ball
+
+> Pickup (Energy Tank); Heals? True
+  * Pickup 17; Major Location? False
+  > Dock to Hive Totem
+      Trivial
 
 ----------------
 Sunchamber Access
 Asset id: 1422133653
 > Dock to Sunchamber; Heals? False
   * Normal Door to Sunchamber/Dock to Sunchamber Access
+  > Dock to Sunchamber Lobby
+      Trivial
 
 > Dock to Sunchamber Lobby; Heals? False
   * Normal Door to Sunchamber Lobby/Dock to Sunchamber Access
+  > Dock to Sunchamber
+      Trivial
 
 ----------------
 Gathering Hall
 Asset id: 1206336453
 > Dock to Watery Hall Access; Heals? False
   * Normal Door to Watery Hall Access/Dock to Gathering Hall
+  > Dock to Gathering Hall Access
+      Trivial
 
 > Dock to Gathering Hall Access; Heals? False
   * Normal Door to Gathering Hall Access/Dock to Gathering Hall
+  > Dock to Watery Hall Access
+      Trivial
+  > Dock to Save Station 2
+      Trivial
+  > Dock to East Atrium
+      Morph Ball
 
 > Dock to Save Station 2; Heals? False
-  * Normal Door to Save Station 2/Dock to Gathering Hall
+  * Missile Blast Shield to Save Station 2/Dock to Gathering Hall
+  > Dock to Gathering Hall Access
+      Trivial
 
 > Dock to East Atrium; Heals? False
   * Normal Door to East Atrium/Dock to Gathering Hall
+  > Pickup (Missile Expansion)
+      Morph Ball Bomb and Space Jump Boots and Morph Ball
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 18; Major Location? False
+  > Dock to East Atrium
+      Trivial
 
 ----------------
 Totem Access
 Asset id: 3934929011
 > Dock to Ruined Gallery; Heals? False
   * Normal Door to Ruined Gallery/Dock to Totem Access
+  > Dock to Hive Totem
+      Trivial
 
 > Dock to Hive Totem; Heals? False
   * Normal Door to Hive Totem/Dock to Totem Access
+  > Dock to Ruined Gallery
+      Trivial
 
 ----------------
 Map Station
 Asset id: 442183190
 > Dock to Ruined Gallery; Heals? False
-  * Normal Door to Ruined Gallery/Dock to Map Station
+  * Missile Blast Shield to Ruined Gallery/Dock to Map Station
 
 ----------------
 Sun Tower Access
 Asset id: 1103925484
 > Dock to Sunchamber; Heals? False
   * Normal Door to Sunchamber/Dock to Sun Tower Access
+  > Dock to Sun Tower
+      Trivial
 
 > Dock to Sun Tower; Heals? False
   * Normal Door to Sun Tower/Dock to Sun Tower Access
+  > Dock to Sunchamber
+      Trivial
 
 ----------------
 Hive Totem
 Asset id: 3358629366
 > Dock to Totem Access; Heals? False
   * Normal Door to Totem Access/Dock to Hive Totem
+  > Dock to Transport Access North
+      Trivial
 
 > Dock to Transport Access North; Heals? False
-  * Normal Door to Transport Access North/Dock to Hive Totem
+  * Missile Blast Shield to Transport Access North/Dock to Hive Totem
+  > Dock to Totem Access
+      Space Jump Boots and After Hive Totem one-time falling blocks
```

Comment:
Is this supposed to be And?

Reply:
yeah after the blocks fallen were removed you need space jump boots else you don't need anything

Reply:
I simplified it a lot by removing the event for falling blocks and putting the hive mecha event as the only way to go back to totem access so it happens once